public/app/plugins/datasource/loki/module.ts

```diff
@@ -1,6 +1,6 @@
 import Datasource from './datasource';
 
-import LokiStartPage from './components/LokiStartPage';
+import LokiStartPage from './components/LokiCheatSheet';
```

Comment:
Nitpick: right now this goes `LokiCheatSheet` ->  `LokiStartPage` ->  `ExploreStartPage`, maybe we can remove the `LokiStartPage` renaming?